src/demo/share/jfc/SwingSet2/resources/swingset_ja.properties

```diff
@@ -202,24 +202,24 @@ ButtonDemo.imagetogglebuttons=\u30A4\u30E1\u30FC\u30B8\u30FB\u30C8\u30B0\u30EB\u
 ButtonDemo.textcheckboxes=\u30C6\u30AD\u30B9\u30C8\u30FB\u30C1\u30A7\u30C3\u30AF\u30FB\u30DC\u30C3\u30AF\u30B9
 ButtonDemo.imagecheckboxes=\u30A4\u30E1\u30FC\u30B8\u30FB\u30C1\u30A7\u30C3\u30AF\u30FB\u30DC\u30C3\u30AF\u30B9
 
-ButtonDemo.button1=1 
+ButtonDemo.button1=1\u0020
 ButtonDemo.button2=2
 ButtonDemo.button3=<html><font size=2 color=red><bold>3</font></html>
 
-ButtonDemo.radio1=\u30DC\u30BF\u30F31 
+ButtonDemo.radio1=\u30DC\u30BF\u30F31\u0020
 ButtonDemo.radio2=\u30DC\u30BF\u30F32
 ButtonDemo.radio3=\u30DC\u30BF\u30F33
 ButtonDemo.radioX=<html><font size=2 color=red>3<bold>(HTML)</bold></font></html>
 
-ButtonDemo.check1=1 
+ButtonDemo.check1=1\u0020
```

Comment:
Used only in JLabels. Since 2 and 3 don't have spaces afterwards (the usages are identical afaict), and since other locales like en don't have the space, it's fine to remove.